resources/views/layouts/sidebars/app.blade.php

```diff
@@ -85,8 +87,8 @@
                                     {!! $child['custom_label'] ?: $child['label'] !!}
                                 </a>
                             </li>
-                            @if($currentCampaign->boosted())
-                                @include('layouts.sidebars._quick-links')
+                            @if ($currentCampaign->boosted())
```

Comment:
You can merge the if and include into a single line by doing `@includeWhen($currentCampaign->boosted(), ' layouts.sidebars._quick_links', [])`